Competition2017/src/competition/subsystems/agitator/AgitatorManagerSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.agitator;
+
+import org.apache.log4j.Logger;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import competition.subsystems.RobotSide;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.logging.RobotAssertionManager;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class AgitatorManagerSubsystem extends BaseSubsystem {
```

Comment:
Just to be consistent with the other ones, consider making this Agitator*s*ManagerSubsystem